ui/src/Live/Multiview.tsx

```diff
@@ -106,18 +109,62 @@ function selectedReducer(old: SelectedCameras, op: SelectOp): SelectedCameras {
  */
 const Multiview = (props: MultiviewProps) => {
   const [searchParams, setSearchParams] = useSearchParams();
-  const theme = useTheme();
 
   const [selected, updateSelected] = useReducer(
     selectedReducer,
     searchParams.has("cams")
-      ? JSON.parse(searchParams.get("cams") || "")
+      ? JSON.parse(searchParams.get("cams") || "") :
+      localStorage.getItem("camsSelected") !== null ?
+        JSON.parse(localStorage.getItem("camsSelected") || "")
       : Array(MAX_CAMERAS).fill(null)
   );
 
+  /**
+   * Save previously selected cameras to local storage.
+   */
+  useEffect(() => {
+    if (searchParams.has("cams")) localStorage.setItem("camsSelected", (searchParams.get("cams") || ""));
+  }, [searchParams]);
+
   const outerRef = React.useRef<HTMLDivElement>(null);
   const layout = LAYOUTS[props.layoutIndex];
 
+  /**
+   * Toggle full screen.
+   */
+  const handleFullScreen = useCallback(() => {
+    if (outerRef.current) {
+      const elem = outerRef.current;
+      //@ts-expect-error another browser (if not bruh)
+      if (document.fullscreenElement || document.webkitFullscreenElement || document.msFullscreenElement) {
```

Comment:
Do we need these `webkitFullScreenElement` and `msFullScreenElement` variants? Chrome and Edge look green here: https://caniuse.com/mdn-api_document_exitfullscreen